src/components/atoms/Wrapper/index.tsx

```diff
@@ -1,15 +1,15 @@
 import styled from "styled-components";
 import { minDevice } from "../../../assets/styles/breakpoints";
 
-import { PropsOptional } from "./interface";
+import { WrapperProps } from "./interface";
 
-const Wrapper = styled.div<PropsOptional>`
+const Wrapper = styled.div<WrapperProps>`
   display: flex;
   flex-direction: ${(props) => props.flexDirection || "row"};
   justify-content: ${(props) => props.justifyContent || "inherit"};
   align-items: ${(props) => props.alignItems || "inherit"};
-  padding: ${(props) => props.padding || "1.6rem"};
-  margin: ${(props) => props.margin || "1.6rem"};
+  padding: ${(props) => props.padding || "0 1.6rem"};
```

Comment:
Have you checked if this change in padding doesn't mess with styles in other places?

Reply:
Wohooooo 🍾